modules/c++/io/tests/serializeTest1.cpp

```diff
@@ -73,7 +73,9 @@ class A : public Serializable
     {
 	std::string toFill;
 	sys::byte b[1];
-	while (true)
+	// Work-around warning C4127: conditional expression is constant;
+	// see https://msdn.microsoft.com/en-us/library/6t66728h.aspx and http://stackoverflow.com/questions/1946445/c-c-how-to-use-the-do-while0-construct-without-compiler-warnings-like-c412
+	for (;;) // while (true)
```

Comment:
See https://blogs.msdn.microsoft.com/vcblog/2016/06/03/c-core-guidelines-checkers-are-now-in-a-single-nuget-package/ (links to http://reviews.llvm.org/diffusion/L/browse/clang-tools-extra/trunk/clang-tidy/cppcoreguidelines/) for one way to get (a lot?) more warnings on Linux.

This (http://stackoverflow.com/a/12247461/8877) SO answer indicates that GCC also has a lot more warnings available than those from -Wall.